packages/fxa-payments-server/src/App.tsx

```diff
@@ -58,7 +58,7 @@ export const App = ({
   return (
     <AppContext.Provider value={appContextValue}>
       <AppLocalizationProvider
-        userLocales={navigator.languages}
+        userLocales={[...navigator.languages]}
```

Comment:
Why do we need to spread `navigator.languages` here?

Reply:
Its a typescript thing, navigator.languages is an array of readonly strings, we need to copy the array to assign it to userLocales.

Reply:
Ah, gotcha.  I wonder if that's still needed if we declare `userLocales` to be readonly also.

Reply:
Declaring as `ReadonlyArray<string>` fixed it. 👍